lighthouse-core/audits/bootup-time.js

```diff
@@ -84,6 +84,7 @@ class BootupTime extends Audit {
    * @return {Promise<LH.Audit.Product>}
    */
   static async audit(artifacts, context) {
+    const runWarnings = artifacts.LighthouseRunWarnings || [];
```

Comment:
shouldn't this always be defined (since part of the base artifacts provided by gatherrunner, but also because specified in `requiredArtifacts`)

Reply:
yeah was just thinking about older fixtures or whatnot, I'll remove